modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_{context}"]
+= Integrating software RAID arrays with {lvms}
```

Comment:
I'd add some caution here to attributes rendering in a heading on the customer portal. It might work, but hard-coding would be a safer route.

Reply:
It could be OK but one would need to access to PV1 and possibly merge the content before hand to check things out. Best to throw caution to the wind and hardcode for headings.

Reply:
:heavy_check_mark:

Reply:
Removed the variable and added the attribute value directly in  the ID.